src/Classes/Area Class.lua

```diff
@@ -96,6 +96,10 @@ function Area:hasPoint(point, y)
            point.y <= self.botRight.y
 end
 
+function Area:isVertical()
+    return self:getHeight() >= self:getWidth()
```

Comment:
I don't really like that we're using `>=` here which implies a square area would be considered vertical. Don't you think we should use `>` instead?